tasks/standupReminder.ts

```diff
@@ -1,17 +1,48 @@
 import { google, calendar_v3 } from "googleapis"
 import { WebClient } from "@slack/client"
 import { peril } from "danger"
+import fetch from "node-fetch"
+import querystring from "querystring"
 
 let googleKey: any = JSON.parse(peril.env.GOOGLE_APPS_PRIVATE_KEY_JSON || "{}")
 const SCOPES = ["https://www.googleapis.com/auth/calendar.readonly"]
 const CALENDAR_ID = peril.env.ON_CALL_CALENDAR_ID || ""
 
 export default async () => {
+  // Backed by Google Calendar
   const events = await retrieveCalendarEvents()
-  await sendMessageForEvents(events)
+  const calendarOnCallStaffEmails = emailsForCalendarEvents(events)
+  await sendMessageForEmails(calendarOnCallStaffEmails)
+
+  // Backed by OpsGenie
+  const opsGenieOnCallStaffEmails = await emailsFromOpsGenie()
+  await sendMessageForEmails(opsGenieOnCallStaffEmails)
```

Comment:
If I'm understanding correctly, this will send two Slack messages? One based on OpsGenie and another based on Google Calendar? Is the idea that we'll only have one running at once? We could alternatively use one call with `sendMessageForEmails(uniq(concat(calendarOnCallStaffEmails, opsGenieOnCallStaffEmails)))`.

Reply:
Love it! :heart: 

The two schedules shouldn't overlap but you're right, as is, this will send two Slack message. I'll pull in your suggested change. Thanks!